server/http.go

```diff
@@ -537,6 +538,7 @@ func handleTokenFunc(srv OIDCServer) http.HandlerFunc {
 			IDToken:      jwt.Encode(),
 			TokenType:    "bearer",
 			RefreshToken: refreshToken,
+			ExpiresIn:    int64(session.DefaultSessionValidityWindow.Seconds()),
```

Comment:
I don't like assuming that we're always using the default validity window, since it ties it to a global variable rather than this server's configuration. Ideally, `CodeToken`, `ClientCredsToken`, and `RefreshToken` would all return the JWT's expiry in it's function signature and the actual configuration of this value lives within those functions.